nodejs/awsx/ec2/vpcTopology.ts

```diff
@@ -113,7 +314,7 @@ ${lastAllocatedIpAddress} > ${lastVpcIpAddress}`);
         return nextCidrBlock;
     }
 
-    private createSubnetsWorker(subnetArgs: x.ec2.VpcSubnetArgs, cidrMask: number) {
+    private createSubnetsWorker(subnetArgs: x.ec2.VpcSubnetArgs, cidrMask: number, currentSubnetIndex: number) {
```

Comment:
needed to pass this value in now as determining ipv6 cidr blocks needs it, and we pushed that logic here from Vpc.